File: lessons/compiled_lang/java/src/e00.branch.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

class E00Branch {
    public static void main(String[] args) throws IOException {
        BufferedReader stdin = new BufferedReader(new InputStreamReader(System.in));
        System.out.print("Would you like a toast? [Yes/No]: ");
        String response = stdin.readLine();

        String message;
        if (response.equals("Yes"))
            message = "That's great!";
        else
            message = "How about a muffin?";

        System.out.println(message);
    }
}

File: lessons/compiled_lang/cs/src/e00.branch.cs
using System;

class E00Branch
{
    static void Main()
    {
        Console.Write("Would you like a toast? [Yes/No]: ");
        string response = Console.ReadLine();

        string message;
        if (response == "Yes")
            message = "That's great!";
        else
            message = "How about a muffin?";

        Console.WriteLine(message);
    }
}
